digital-deck-server/services/cardShuffleService.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+    /**
+     * Shuffles the cards in the deck
+     * 
+     * @param cards The deck of cards to be drawn from
+     * @return {Array} The json containing the ids cards currently in the player's hand and the current deck
+     */
+    shuffleCards: function(cards) {
+      let shuffledCards = cards.slice();
+      if (shuffledCards.length>1) {
+        for (let i = cards.length - 1; i > 0; i--) {
```

Comment:
I am curious as to why just randomly swap with only remaining values and not the whole array? ``` Math.floor(Math.random() * i); ``` . Talking about using ```i``` here. Is it better this way?